### ISSUE TITLE
[BUG] [FEATURE] 

### ISSUE THREAD

**Issue by dvnicolasdh** [keitaroinc/ckanext-s3filestore#7]
With a compiled version 2.9.4 of CKAN
- https://github.com/ckan/ckan

and complied version v1.0.0 of ckanext-s3filestore
- https://github.com/keitaroinc/ckanext-s3filestore

and configured to store on a Minio version: 2020-09-26T03:44:56Z (same behavior with Minio version: RELEASE.2021-11-24T23-19-33Z)

* Uploading/Downloading of Dataset Ressources through CKAN does store/retreive in S3 (MinIO).
* But **removal** or **deletion** of a Dataset's Ressource in CKAN doesn't remove that ressource from S3 (MinIO).

**Expected behaviour**
One could expect deletion mechanism to have the actual ressource deleted from storage, including S3 (MinIO) .

**Logs**
There are no "errror" messages...

On another ckanext-s3filestore's archived github project's "bug/feature" report/response:
-    bug/feature request: https://github.com/okfn/ckanext-s3filestore/issues/13

seem to point to "clear_key" definition and usage in the "[uploader.py](https://github.com/keitaroinc/ckanext-s3filestore/blob/main/ckanext/s3filestore/uploader.py)".

Any thought on this ?

**Comment by HristijanVilos** [comment]
Yes this is a known issue and could be considered as a feature since CKAN local storage works the same i.e. it does not delete resources from the local file system. We will take this in consideration and implement this feature.
